src/main/java/com/ruchi/leetcode/Palindrome.java

```diff
@@ -0,0 +1,24 @@
+package com.ruchi.leetcode;
+
+import javax.sound.midi.Soundbank;
+
+public class Palindrome {
+    public Boolean checkPalindrome(String str)
+    {
+        Boolean result = true;
+        char[] chars = str.toCharArray();
+        for(int i=0,j=chars.length-1;i<j;i++,j--)
+        {
+          if(chars[i]==chars[j])
+          {
+              result =true;
+          }
+          else
+          {
+              result =false;
+              return result;
+          }
```

Comment:
reverse the condition if not equals and return false [why to check whole string if it failed on first match

Reply:
[changes done](url)